docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[troubleshooting]]
+== Troubleshooting
+
+Below are some resources and tips for troubleshooting and debugging the
+python agent.
+
+[float]
+[[easy-fixes]]
+=== Easy Fixes
+
+Before you try anything else, go through the following sections to ensure that
+the agent is configured correctly. This is not an exhaustive list, but rather
+a list of common problems that users run into.
+
+[float]
+[[debug-mode]]
+==== Debug Mode
+
+Most frameworks support a debug mode. Generally, this mode is intended for
+non-prod environments and provides detailed error messages and logging of
+potentially sensitive data. Because of these security issues, the agent will
+not collect traces if the app is in debug mode by default.
+
+You can override this behavior with the <<config-debug,`DEBUG`>> configuration.
+
+Note that configuration of the agent should occur before creation of any
+`ElasticAPM` objects:
+
+[source,python]
+----
+app = Flask(__name__)
+app.config["ELASTIC_APM"] = {"DEBUG": True}
+apm = ElasticAPM(app, service_name="flask-app")
+----
+
+[float]
+[[psutil-metrics]]
+==== `psutil` for Metrics
+
+To get CPU and system metrics, `psutil` must be installed. The agent should
```

Comment:
This is only true for non-Linux systems. On Linux, we get most metrics by reading a bunch of files in `/proc`